features/aa/aa_features/sponsorship.feature

```diff
@@ -66,13 +65,16 @@ Feature:
     And I should see "Second Orphan"
     And I fill in Sponsorship Start Date for "First Orphan" with "2014-01-31"
     When I click the "Sponsor this orphan" link for orphan "First Orphan"
+    Then I should be on the Link to Orphan page for sponsor "First Sponsor"
+    And I should see "Return to Sponsor page"
+    And I should see "Sponsorship link was successfully created for First Orphan"
+    And I fill in Sponsorship Start Date for "Second Orphan" with "2014-01-31"
+    When I click the "Sponsor this orphan" link for orphan "Second Orphan"
+    Then I should be on the Link to Orphan page for sponsor "First Sponsor"
```

Comment:
shouldn't we set the `sponsor` with `Requested orphan count=2`? And after the `Second Orphan` is sponsored, the sponsor orphan requests are fulfilled and it redirects to `sponsor#index`?